EnglishFilter/sections/whitelist.txt

```diff
@@ -214,6 +214,8 @@ copypastecharacter.com#$##contents { visibility: visible!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/55620
 @@||edge.microsoft.com/translate/auth
 @@||api.cognitive.microsofttranslator.com/translate?
+! https://github.com/AdguardTeam/AdguardFilters/issues/55576
```

Comment:
to JP filter, I guess

Reply:
but `.single-ads` is from the base filter?

Reply:
Or rather EasyList. @ryanbr